### PR TITLE
Add latest tag to docker image

### PR DESCRIPTION
**Pull request by rakanalh**
### Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Dependency upgrade (A change in substrate or any 3rd party crate version)

### Migrations and Hooks

- [ ] This change requires a runtime migration.
- [ ] Modifies `on_initialize`
- [ ] Modifies `on_finalize`

### Checklist

- [ ] Change has been tested locally.
- [ ] Change adds / updates tests.
- [ ] Changelog doc updated.

### CODE REVIEW

File: .github/workflows/publish-docker-image-to-dockerhub.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ jobs:
           docker pull $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA
           docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:${GITHUB_REF##*/}
           docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $ECR_REGISTRY/$ECR_REPOSITORY:${GITHUB_REF##*/}
+          docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:latest
```
Comment (upalinski) [inline .github/workflows/publish-docker-image-to-dockerhub.yaml:31]: you added tag to ECR repository but we expected it to be pushed to dockerhub, so it should be added on line 42 i guess...

Comment (rakanalh) [reply .github/workflows/publish-docker-image-to-dockerhub.yaml:31]: No it's creating a tag `$DOCKERHUB_REPOSITORY/$PACKAGE_NAME:latest` based on `$ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA`

**Review by MRamanenkau — APPROVED** [review]

File: .github/workflows/publish-docker-image-to-dockerhub.yaml (hunk 1 above)
Comment (upalinski) [reply .github/workflows/publish-docker-image-to-dockerhub.yaml:31]: hm. shouldn't you push $DOCKERHUB_REPOSITORY/$PACKAGE_NAME:latest in `Publish image to DockerHub` step?

Comment (rakanalh) [reply .github/workflows/publish-docker-image-to-dockerhub.yaml:31]: There are 2 steps here:
`docker tag Image:Tag OtherImage:OtherTag`

Which is what we do here. 

Down the line, we push the `OtherImage:OtherTag`

Comment (upalinski) [reply .github/workflows/publish-docker-image-to-dockerhub.yaml:31]: ok

**Review by MRamanenkau — APPROVED** [review]

**Review by upalinski — APPROVED** [review]

File: .github/workflows/publish-docker-image-to-dockerhub.yaml (hunk 1 above)
Comment (rakanalh) [reply .github/workflows/publish-docker-image-to-dockerhub.yaml:31]: I realized what you meant afterwards. Fixed.

**Pull request merged**